src/components/predictive-layer.vue

```diff
@@ -14,8 +14,22 @@
           : 'x-row x-row--gap-06 x-row--align-start'
       "
     >
+      <IdentifierResults
+        #default="{ identifierResult }"
+        :animation="suggestionsAnimation"
+        class="x-row__item x-row__item--span-12 x-list x-list--gap-04"
```

Comment:
The gap should be 8px, `x-list--gap-03`, like in the `QuerySuggestions`.

![image](https://user-images.githubusercontent.com/68222542/143564348-acd2c029-3390-4b5b-a7f3-6288d7daae30.png)

Reply:
<img width="345" alt="image" src="https://user-images.githubusercontent.com/3502075/143584420-5a1dead1-d504-4619-82d5-de3b86e00996.png">

I see 16px

Reply:
And 12px 😆 

<img width="837" alt="image" src="https://user-images.githubusercontent.com/3502075/143584669-827585ca-f043-4379-995d-9f51cef76925.png">

Reply:
javi is Right. As I always say, in these cases the designs are illustrative, but we have to keep the consistency in the setup.

so change to `gap--03`